hack/util.sh

```diff
@@ -92,9 +92,19 @@ function wait_for_url {
   wait=${3:-0.2}
   times=${4:-10}
 
+  CURL_CERT=${CURL_CERT:-}
+  CURL_KEY=${CURL_KEY:-}
+  clientcert_args=""
+
+  if [ -n "${CURL_CERT}" ]; then
+   if [ -n "${CURL_KEY}" ]; then
+     clientcert_args="--cert ${CURL_CERT} --key ${CURL_KEY}"
+   fi
+  fi
+
   set +e
   for i in $(seq 1 $times); do
-    out=$(curl -fs $url 2>/dev/null)
+    out=$(curl ${clientcert_args} -fs $url 2>/dev/null)
```

Comment:
Also add `${clientcert_args}` below to the final `curl` invocation so we don't surface a misleading error message